### ISSUE TITLE
i18n for notification widget

### ISSUE THREAD

**Issue by proDOOMman** [novuhq/novu#580]
## What?
Widget localization ("Notifications", "Mark all as read")

## Why?
Novu users speaks different languages

**Comment by scopsy** [comment]
@proDOOMman Thank you for opening this issue. Are you using the react component for this? I think the best would be to provide an object with the entire texts contents of the notification center and than customers can override it using their preferred i18n library. What do you think about this approach?

**Comment by proDOOMman** [comment]
Yes, I use react component.
> provide an object with the entire texts contents of the notification center and than customers can override it using their preferred i18n library

I agree. Great idea.

**Comment by djabarovgeorge** [comment]
Hey @scopsy @proDOOMman,
Currently, we support custom `header` and `footer`, so you could override the header component and design it as you please. Is this approach will answer your needs?

**Comment by WinterCore** [comment]
Hey. can I work on this :smile:?

**Comment by ainouzgali** [comment]
@WinterCore great! Thank you!

**Comment by WinterCore** [comment]
@scopsy

> I think the best would be to provide an object with the entire texts contents of the notification center and than customers can override it using their preferred i18n library. What do you think about this approach?

Are you talking about something like the following?
```ts
const labels = {
  notifications: "Notifications",
  markAllAsRead: "Mark all as read",
  poweredBy: "Powered By",
};

<NovuProvider subscriberId="<USER_ID>" applicationIdentifier="<APP_ID>" labels={labels}>
  <PopoverNotificationCenter colorScheme="light" onNotificationClick={onNotificationClick}>
    {({ unseenCount }) => <NotificationBell unseenCount={unseenCount} />}
  </PopoverNotificationCenter>
</NovuProvider>
```

**Comment by scopsy** [comment]
Merged at #854
